src/ducks/mine-filter-ui.ts

```diff
@@ -1,42 +1,53 @@
 import {OrNothing} from "../utils/types/types";
-import {MineFilter} from "./mine-filter";
+import {LagretFilter} from "./lagretFilter";
 
 // Actions
 export const VELG_MINE_FILTER = 'minefilter_velg/VELG_MINE_FILTER';
 export const MARKER_MINE_FILTER = 'minefilter_velg/MARKER_MINE_FILTER';
 export const AVMARKER_MINE_FILTER = 'minefilter_velg/AVMARKER_MINE_FILTER';
 export const AVMARKER_SISTE_VALGT_FILTER = 'minefilter_velg/AVMARKER_SISTE_VALGT_FILTER';
 
+export const MARKER_VEILEDER_GRUPPE = 'minefilter_velg/MARKER_VEILEDER_GRUPPE';
+export const AVMARKER_VEILEDER_GRUPPE = 'minefilter_velg/AVMARKER_VEILEDER_GRUPPE';
+
 export const APEN_LAGRE_FILTER_MODAL = 'minefilter_velg/APEN_MINE_FILTER_MODAL';
 export const LUKK_LAGRE_FILTER_MODAL = 'minefilter_velg/LUKK_MINE_FILTER_MODAL';
 
 
-export interface MineFilterUIState {
-    valgtMineFilter: OrNothing<MineFilter>;
-    sisteValgteMineFilter: OrNothing<number>
+export interface FilterUIState {
+    valgtFilter: OrNothing<LagretFilter>;
+    valgtVeilederGruppe: OrNothing<LagretFilter>;
```

Comment:
is it necessary to have two identical variables?

Reply:
Probably there is some other way. I introduced _valgtVeilederGruppe_ for veileder oversikt and enhetens oversikt so that I can fix some bug when we have two _lagrede filter_ with the same filter selection. So _valgtVeilederGruppe_ in veileder oversikt is used to decide which veileder grupppe is selected based on _filter id_, and similar logic is for enhetens oversikt...only difference is that in enhetens oversikt _veiledergruppe_ is just part of lagrede filter, and in Veileder oversikt that is whole filter. So to simplify, _valgtFilter_ would work fine for veileder oversikt, but not for enhetens oversikt, since that variable points to whole filter, and not just to veileder gruppe, so that is reason for new variable.